athena/llm_core/llm_core/models/providers/ollama_model_config.py

```
@@ -0,0 +1,29 @@
+
+    PROVIDER: ClassVar[str] = "ollama"
+    ENUM: ClassVar[type] = OllamaModel
+    KW_REMAP: ClassVar[dict[str, str]] = {}
+
+    provider: Literal["ollama"] = Field("ollama", const=True)
+    model_name: OllamaModel = Field(
+        ...,
+        description="Ollama model tag (enum value).",
+    )
+
+    def get_model(self) -> BaseLanguageModel:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)